pkg/mflag/flag.go

```diff
@@ -1058,7 +1058,7 @@ func (fs *FlagSet) Parse(arguments []string) error {
 		case ContinueOnError:
 			return err
 		case ExitOnError:
-			os.Exit(2)
+			os.Exit(125)
```

Comment:
Why?  125 means the command failed in a lot of situations (Or ECANCELLED in errno).  Exit 2 means either nothing (or file does not exist in ERRNO).

I think 125 is a more correct error.